module/userExtension/helper.js

```diff
@@ -227,6 +227,20 @@ module.exports = class UserExtensionHelper {
         const allProgramIds = new Set();
         const allUserIds = new Set();
 
+        //delete invalid program operations from csv data 
+        for(let i= userRolesCSVData.length -1; i>=0; i--){
+          const userRole = gen.utils.valueParser(userRolesCSVData[i]);
+          if (!userRole.programOperation || !(gen.utils.upperCase(userRole.programOperation) == messageConstants.common.REMOVE_OPERATION || gen.utils.upperCase(userRole.programOperation) == messageConstants.common.ADD_OPERATION )) {
```

Comment:
@borkarsaish65 I think convertion of operation to upper case we can handle here itself

Reply:
fixed now